docs/tuf-spec.txt

```diff
@@ -1008,6 +1008,23 @@ Version 1.0 (Draft)
    ensure that outdated clients remain able to update, without requiring all 
    previous root keys to be kept to sign new root.json metadata.
 
+   The exact steps to perform the root.json migration should be as follows:
+     
+   1. The client starts at version N of root.json
+
+   2. The client downloads version N + 1 of root.json
+
+   3. The client uses the keys and threshold defined in version N to
+      validate the signatures of N + 1
+
+   4. The client flushes all knowledge of root.json verion N then uses the
+      updated keys and threshold to validate the signatures of N + 1
```

Comment:
(Should this be N + 2?)

Either way, I this this is probably slightly wrong, and may leave you in a state where you have no trusted root metadata.

I would say that 4 should be removed and replaced by:
4. The client uses the keys and threshold defined in version N + 1 to validate the signatures on N + 1.

4.5. If both validations have succeeded, _then_ flush all knowledge of root.json version N and heretofore use N + 1's keys and thresholds to validate further versions.